cli/azd/cmd/sets.go

```diff
@@ -90,10 +91,17 @@ func newAzdContext() (*azdcontext.AzdContext, error) {
 	return azdCtx, nil
 }
 
-func newCredential() (azcore.TokenCredential, error) {
-	credential, err := azidentity.NewAzureCLICredential(nil)
+func newCredential(authManager *auth.Manager) (azcore.TokenCredential, error) {
+	// TODO(ellismg): I guess we should inject this context object at some point?
+	ctx := context.TODO()
```

Comment:
`context` is available now with latest changes in `main`

Reply:
Fixed!